lib/src/generator/templates.aot_renderers_for_html.dart

```diff
@@ -1137,6 +1137,29 @@ String renderLibrary(LibraryTemplateData context0) {
   return buffer.toString();
 }
 
+String renderLibraryRedirect(LibraryTemplateData context0) {
+  final buffer = StringBuffer();
+  buffer.write('''<!DOCTYPE html>
+<html lang="en">
```

Comment:
Might could add a `canonical` tag, too? For the search engines?

Reply:
https://developers.google.com/search/docs/crawling-indexing/consolidate-duplicate-urls

Reply:
So good! Thanks Kevin, I love this. Done.